src/main/webapp/app/exercises/shared/plagiarism/plagiarism-split-view/text-submission-viewer/text-submission-viewer.component.ts

```diff
@@ -153,12 +153,16 @@ export class TextSubmissionViewerComponent implements OnChanges {
 
             const idxColumnFrom = from.column - 1 + offsets[idxLineFrom];
 
-            rows[idxLineFrom] = this.insertToken(rows[idxLineFrom], this.tokenStart, idxColumnFrom);
+            if (rows[idxLineFrom]) {
+                rows[idxLineFrom] = this.insertToken(rows[idxLineFrom], this.tokenStart, idxColumnFrom);
+            }
             offsets[idxLineFrom] += this.tokenStart.length;
```

Comment:
We probably shouldn't increase the offset if we don't insert anything, or should we? (I'm not sure.)